community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Modernizing the conference management backend of the World Affairs Conference'
+description: 'Putting the spotlight on Samuel Martineau, a software engineer based out of Ontario, Canada replacing the backend of a conference management system with Xata.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cs-wac-cover.jpg
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 05-08-2024
+tags: ['community']
+published: true
+slug: community-spotlight-conference-backend-world-affairs-conference
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cs-wac-cover.jpg
+---
+
+[Samuel Martineau](https://github.com/Samuel-Martineau) has not even graduated high school yet and is already racking up an impressive list of professional accomplishments. From being on national TV for his [wall climbing robot](https://github.com/Samuel-Martineau/memomath), to leading the team and architectural design for the new management system of the worlds oldest student run conference, the [World Affairs Conference](https://worldaffairscon.org/). The conference brings in over 4,000 of the worlds brightest minds from 25 countries in 65 schools internationally. As Samuel gets ready to graduate from the [Upper Canada College](https://en.wikipedia.org/wiki/Upper_Canada_College), he looks to take his talents to the world of mechatronics, pursuing a degree in mechanical engineering at a Canadian university.
```

Comment:
The international statistics date from the pandemic back when we were virtual. Though we intend on expending back to live broadcasting starting next year, we focused on the Greater Toronto Area this year.

Also, though we would dream to be able to accommodate 4000 students, our facilities are already filled at the 1000 mark.

Reply:
My school would prefer if the link to Upper Canada College was our official https://www.ucc.on.ca/ rather than the third party Wikipedia link.

Reply:
Updated ✅